src/PHPCR/Util/QOM/Sql2Scanner.php

```diff
@@ -160,11 +160,12 @@ protected function scan($sql2)
         }
 
         $regexp = '';
```

Comment:
i wonder why we initialized this variable like this, it made no sense. just delete the line above